pynance/dkb.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import csv
+import datetime
+import re
+
+import numpy as np
+
+from .textimporter import CsvFileDescription
+
+# DKB definitions
+# for most solutions, see
+# https://github.com/hamvocke/dkb2homebank/blob/master/dkb2homebank.py
+# (MIT licenced)
+
+
+class DKBFormatters():
+    """
+        contains all the static functions needed to convert a string
+        as given from a DKB CSV file to all the types needed in COLUMNS
+    """
+
+    @classmethod
+    def to_datetime64(cls, datestring):
+        # TODO: there is probably a better way to convert it
+        date = datetime.datetime.strptime(datestring, "%d.%m.%Y")
+        return np.datetime64(date.strftime('%Y-%m-%d'))
+
+    @classmethod
+    def to_string(cls, anystring):
+        # remove trailing whitespace
+        return anystring.strip()
+
+    @classmethod
+    def to_float64(cls, numberstring):
+        if not numberstring:
+            return np.nan
+        else:
+            # TODO: use some builtin package, maybe locale for this
+            return float(numberstring.replace(".", "").replace(",", "."))
```

Comment:
Not that we use locale [elsewhere](https://github.com/Sh4pe/pynance/blob/master/pynance/parser_test.py#L35), shouldn't we use it here as well?

Reply:
hm I'm not sure about using locale. Downside:
* it's more code
* it has dependencies (see that language pack issue)
* it's a black box
* it tends to have behaviour that depends on the machine (in Windows the locale is called "German", on linux sometimes "de_DE", sometimes "de_DE.utf8") In the tests I find this ok, since it's the purpose of the test to find unexpected behaviour, but here not

I think we can search for other options to generalize string to number, but this is just for DKB files, so I would leave it like this at the moment until we want to generalize this for other formats.

What do you think?